src/write_influxdb_udp.c

```diff
@@ -0,0 +1,729 @@
+/**
+ * collectd - src/write_influxdb_udp.c
+ * Copyright (C) 2007-2009  Florian octo Forster
+ * Copyright (C) 2009       Aman Gupta
+ * Copyright (C) 2019       Carlos Peon Costa
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; only version 2 of the License is applicable.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ *
+ * Authors:
+ *   Florian octo Forster <octo at collectd.org>
+ *   Aman Gupta <aman at tmm1.net>
+ *   Carlos Peon Costa <carlospeon at gmail.com>
+ **/
+
+#include "collectd.h"
+
+#include "plugin.h"
+#include "utils/common/common.h"
+#include "utils_cache.h"
+#include "utils_complain.h"
+
+#if HAVE_NETDB_H
+#include <netdb.h>
+#endif
+#if HAVE_NETINET_IN_H
+#include <netinet/in.h>
+#endif
+#if HAVE_ARPA_INET_H
+#include <arpa/inet.h>
+#endif
+#if HAVE_POLL_H
+#include <poll.h>
+#endif
+#if HAVE_NET_IF_H
+#include <net/if.h>
+#endif
+
+struct sockent_client {
+  int fd;
+  struct sockaddr_storage *addr;
+  socklen_t addrlen;
+  cdtime_t next_resolve_reconnect;
+  cdtime_t resolve_interval;
+  struct sockaddr_storage *bind_addr;
+};
+
+struct sockent_server {
+  int *fd;
+  size_t fd_num;
+};
+
+typedef struct sockent {
+#define SOCKENT_TYPE_CLIENT 1
```

Comment:
As I see, you use only CLIENT.

Please remove all dead/unused code, related to SERVER.

Reply:
Thank you for review.

I think it's done.

CI build failed with:
`checking whether build environment is sane... configure: error: newly created file is older than distributed files!
Check your system clock
The command "./configure" exited with 1.`

Reply:
>CI build failed with:

That is known issue (for me), unfortunately nobody cares.
If you want, you can do `git commit --amend` + `git push -f ... `.

Reply:
Thank you for the CI workaround :)